nimterop/exprparser.nim

```diff
@@ -548,7 +548,15 @@ proc processTSNode(gState: State, node: TSNode, typeofNode: var PNode): PNode =
     # Input -> true, false
     # Output -> true, false
     result = gState.parseString(node.val)
-  of "type_descriptor", "sized_type_specifier":
+  of "type_descriptor":
+    let pointerDecl = node.anyChildInTree("abstract_pointer_declarator")
```

Comment:
This could be any number of pointers, need to count them and generate code accordingly. See how pointers are handled in ast2, can use the same strategy here too, perhaps reuse the code as well.